src/main/java/com/alkemy/ong/web/controller/MemberController.java

```diff
@@ -0,0 +1,105 @@
+package com.alkemy.ong.web.controller;
+
+import com.alkemy.ong.domain.members.Member;
+import com.alkemy.ong.domain.members.MemberService;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/members")
+public class MemberController {
+
+    private final MemberService memberService;
+
+    @Autowired
+    public MemberController(MemberService memberService) {
+        this.memberService = memberService;
+    }
+
+    @PostMapping
+    public ResponseEntity<MemberDTO> save(@Valid @RequestBody MemberDTO dto) {
+        Member memberSaved = this.memberService.save(this.toModel(dto));
```

Comment:
el this no es necesario. Va solo en el constructor para diferenciar las variables que le llegan por parámetro de las definidas en la clase.